Awful.apk/src/main/java/com/ferg/awfulapp/thread/ForumParsing.kt

```diff
@@ -121,13 +121,20 @@ class PostParseTask(
             put(IS_PLAT, postData.hasDescendantWithClass("platinum").sqlBool)
             put(ROLE, getRole())
 
-            // grab the custom title, and also the avatar if there is one
+            // grab the custom title, and also avatar and alternate avatar if there are any
             postData.selectFirst(".title")!!
                 .also { put(AVATAR_TEXT, it.text()) }
-                .selectFirst("img")
-                ?.let {
-                    tryConvertToHttps(it)
-                    put(AVATAR, it.attr("src"))
+                .select("img")
+                .forEachIndexed { index, image ->
```

Comment:
Wouldn't that keep overwriting A_S if there are more than 2 images in the user title?

Reply:
It's the next line that makes sure it doesn't -- in a terrible hack, we only take action if the index is 0 or 1, so just the first and second images are processed.

Reply:
Latest commit uses take(2), clarifying the intent here.